orchagent/pfcwdorch.cpp

```diff
@@ -759,6 +759,11 @@ void PfcWdSwOrch<DropHandler, ForwardHandler>::doTask(Consumer& consumer)
 {
     PfcWdOrch<DropHandler, ForwardHandler>::doTask(consumer);
 
+    if (!gPortsOrch->allPortsReady())
+    {
+        return;
+    }
+
```

Comment:
This check is redundant as calling PfcWdOrch<DropHandler, ForwardHandler>::doTask(consumer) on line 760 has this check (see line 57)

Reply:
Does it cause a warm-reboot issue other than the ERR syslog message? As the entry is not removed from m_toSync, it will succeed starting PFC watchdog storm action in the next iteration of doTask().

Reply:
Yes, storm action starts successfully but on later iteration. However, test case #2 catches this error in log analysis

Reply:
Can you add some comments to the codes explaining why you added the check?